scripts/fetch_feed_cache.py

```
@@ -0,0 +1,97 @@
+# few screens of scroll before the user is likely back online. The live feed
+# holds up to 500, so this is a head slice, not the whole thing.
+PAGES = 3
+LIMIT = 50
+TIMEOUT_S = 20
+
+
+def fetch_page(platform_param, page):
+    url = f"{BACKEND}/v1/feed?page={page}&limit={LIMIT}"
+    if platform_param:
+        url += f"&platform={platform_param}"
+    req = urllib.request.Request(url, headers={"User-Agent": "feed-cache-publisher/1.0"})
```

Ruff (0.15.17)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)